src/avatar/Avatar.js

```diff
@@ -264,6 +236,19 @@ Avatar.propTypes = {
   }),
 };
 
-Avatar.defaultProps = defaultProps;
+Avatar.defaultProps = {
+  showEditButton: false,
+  onEditPress: null,
+  width: 34,
+  height: 34,
```

Comment:
Maybe use your `DEFAULT_SIZES` object here, to have only one source for the sizes

Reply:
I will do that @xavier-villelegier 😀typescript 😍